src/title.rs

```diff
@@ -1,25 +1,76 @@
+#![warn(missing_docs)]
```

Comment:
If we want to go with this, let's add this to the higher modules rather than just here I would say.

Reply:
In the long run I agree, but right now if we put it in `lib.rs`, clippy will complain in every pipeline.

How should we handle this? Progressively moving the warning as the documentation grows or just add it when everything is documented?

I think moving it progressively avoids regressions (eg. someone adding a feature in title and not documenting).

Reply:
I agree with this logic use tools to help drive down the missing docs and keep it there.

Slapping this on lib.rs = 359 violations, so we're a ways off yet, but I'd bet there's a few modules that are either fully documented already or close to it that we could add it to.